timeless/restaurants/models.py

```diff
@@ -133,11 +133,6 @@ class Reservation(TimestampsMixin, DB.Model):
 
     tables = DB.relationship("TableReservation", back_populates="reservation")
 
-    @validate_required("start_time", "end_time", "num_of_persons", "comment",
-                       "status")
-    def __init__(self, **kwargs):
-        super(Reservation, self).__init__(**kwargs)
-
```

Comment:
@vladarefiev why did you remove the validation rules? :D

Reply:
@amihaiemil It's useless, we dont need it anywhere,  forms and factories does not work with it.
Db models can validate it by themself, and we should validate it in forms, if we use forms and templates.